aspnetcore/release-notes/aspnetcore-9/includes/blazor.md

```diff
@@ -232,6 +223,6 @@ The <xref:Microsoft.AspNetCore.Components.Forms.InputNumber%601> component now s
 
 <!-- UPDATE 10.0 Confirm or update -->
 
-Support for multiple Blazor Web Apps per server project will be considered for .NET 10 (November, 2025).
+Support for multiple Blazor Web Apps per server project will be considered for .NET 10 (November  2025).
```

Comment:
Typically, I'd expect the comma. Let's consider rolling it into the sentence. It reads a bit better ...

```suggestion
Support for multiple Blazor Web Apps per server project is under consideration for .NET 10 in late 2025.
```

Reply:
I would just change "(November 2025)" to "(late 2025)"